### DISCUSSION TITLE
Suricata Process

### DISCUSSION THREAD

**Discussion by techn00** [Security-Onion-Solutions/securityonion#7325] [Unsupported Versions] [answered]
How much security process I should put here ? Any recommendations
I have server with 32GB RAM, 8vCPU, 1TB drive.

![image](https://user-images.githubusercontent.com/100316438/155748746-f0126352-1913-4815-bedc-bb47313f9346.png)

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/hardware.html#cpu:
> [Suricata](https://docs.securityonion.net/en/2.3/suricata.html#suricata) and [Zeek](https://docs.securityonion.net/en/2.3/zeek.html#zeek) are very CPU intensive. The more traffic you are monitoring, the more CPU cores you’ll need. A very rough ballpark estimate would be 200Mbps per [Suricata](https://docs.securityonion.net/en/2.3/suricata.html#suricata) worker or [Zeek](https://docs.securityonion.net/en/2.3/zeek.html#zeek) worker. So if you have a fully saturated 1Gbps link and are running [Suricata](https://docs.securityonion.net/en/2.3/suricata.html#suricata) and [Zeek](https://docs.securityonion.net/en/2.3/zeek.html#zeek), then you’ll want at least 5 [Suricata](https://docs.securityonion.net/en/2.3/suricata.html#suricata) instances and 5 [Zeek](https://docs.securityonion.net/en/2.3/zeek.html#zeek) workers, which means you’ll need at least 10 CPU cores for [Suricata](https://docs.securityonion.net/en/2.3/suricata.html#suricata) and [Zeek](https://docs.securityonion.net/en/2.3/zeek.html#zeek) with additional CPU cores for [Stenographer](https://docs.securityonion.net/en/2.3/stenographer.html#stenographer) and/or other services.
